### PR TITLE
Added few faqs

### PR DESCRIPTION
**Pull request by kartikeya2609**
Added few faqs

Thanks for contributing to the Harness Developer Hub! Our code owners will review your submission.

### CODE REVIEW

**Review by brian-f-harness — APPROVED** [review]
Edited and approved for PL

**Review by brian-f-harness — APPROVED** [review]

File: kb/platform/harness-platform-faqs.md [hunk 1]
```diff
@@ -82,6 +82,15 @@ Resource groups are an RBAC component that defines the objects that a user or se
 
 For more information, go to [Add resource groups](/docs/platform/role-based-access-control/add-resource-groups).
 
+### Does Harness support provision users and user groups via GitHub?
```
Comment (brian-f-harness) [inline kb/platform/harness-platform-faqs.md:85]:
```suggestion
### Does Harness support provisioning users and user groups via GitHub?
```

**Review by brian-f-harness — APPROVED** [review]

File: kb/continuous-delivery/continuous-delivery-faqs.md [hunk 2]
```diff
@@ -7546,3 +7546,15 @@ Any variable used as `<+input>` will be asked to provide a value at the runtime.
 #### Can you tell me which steps for a custom stage run directly on the delegate?
 Usually all actual task executions occur on delegate. You can add shell script steps to execute the command on delegate.
 
+#### Can user delete the service without removing the active instances?
+Yes, User can delete the services that have active instance by enabling the Force delete option in the account settings.
```
Comment (SushrutHarness) [inline kb/continuous-delivery/continuous-delivery-faqs.md:7550]:
```suggestion
#### Can user delete a service without removing its active instances?
Yes, a user can delete services that have active instances by enabling the Force delete option in the account settings.
```

File: kb/continuous-delivery/continuous-delivery-faqs.md [hunk 3]
```diff
@@ -7546,3 +7546,15 @@ Any variable used as `<+input>` will be asked to provide a value at the runtime.
...
+
+#### Is it possible to update the exsiting Jira from harness pipeline?
+Yes, You can update the existing Jira by using the Jira Update step.
```
Comment (SushrutHarness) [inline kb/continuous-delivery/continuous-delivery-faqs.md:7553]:
```suggestion
#### Is it possible to update an existing Jira ticket from a Harness pipeline?
Yes, you can update an existing Jira ticket by using the Jira Update step.
```

File: kb/continuous-delivery/continuous-delivery-faqs.md [hunk 4]
```diff
@@ -7546,3 +7546,15 @@ Any variable used as `<+input>` will be asked to provide a value at the runtime.
...
+
+#### Is stage output host looping startegy is supported with the custom stage pipeline?
+No, Stage output host looping startegy is only supported with CD stage.
+Doc for reference: https://developer.harness.io/docs/platform/pipelines/looping-strategies/looping-strategies-matrix-repeat-and-parallelism/#repeat-on-multiple-target-hosts
```
Comment (SushrutHarness) [inline kb/continuous-delivery/continuous-delivery-faqs.md:7557]:
```suggestion
#### Is a stage output host looping strategy supported with a custom stage pipeline?
No, Stage output host looping strategy is only supported within a CD stage.
For more information, go to [Looping Strategies](https://developer.harness.io/docs/platform/pipelines/looping-strategies/looping-strategies-matrix-repeat-and-parallelism/#repeat-on-multiple-target-hosts)
```

File: kb/continuous-delivery/continuous-delivery-faqs.md [hunk 5]
```diff
@@ -7546,3 +7546,15 @@ Any variable used as `<+input>` will be asked to provide a value at the runtime.
...
+
+#### Can a user create/update a confluence page from harness step?
+No, As per the current design, there's no native step to create/update a confluence page.
```
Comment (SushrutHarness) [inline kb/continuous-delivery/continuous-delivery-faqs.md:7560]:
```suggestion
#### Can a user create/update a confluence page from a Harness step?
No, currently there is no native step to create or update a confluence page.
```

**Review by SushrutHarness — APPROVED** [review]
LGTM on the CD side

**Review by brian-f-harness — APPROVED** [review]

**Comment by brian-f-harness** [comment]
Fixed merge conflict. Reviewed and approved by @SushrutHarness for CD. I previously approved for PL content. Will merge PR when build succeeds.

**Pull request merged**